docs/guided-remediation.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: page
+title: Guided Remediation
+permalink: /experimental/guided-remediation/
+parent: Experimental Features
+nav_order: 1
+---
+
+# Guided Remediation
+
+Experimental
+{: .label }
+
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+The `osv-scanner fix` subcommand leverages [deps.dev](https://deps.dev) to provide automated and guided remediation of vulnerabilities in your project's dependencies by suggesting upgrades to dependencies.
+
+Currently, only npm `package.json` manifests and `package-lock.json` lockfiles are supported.
+
+{: .note }
+This feature is experimental and may change in future updates.
+
+## Basic Usage: Interactive Mode
```

Comment:
Thinking more, we should start with the CLI usage, and then transition to the interactive mode as an alternative for power users. 

Starting with a heavyweight interactive mode might scare off readers.

Reply:
Moved non-interactive to be 'basic usage', with some forward links to the sections describing in-place/relock strategies.